packages/editor/src/components/url-popover/index.js

```diff
@@ -26,8 +26,9 @@ class URLPopover extends Component {
 		const {
 			children,
 			renderSettings,
-			position,
-			onClickOutside,
+			onClose,
+			onClickOutside = onClose,
```

Comment:
Do we need to have both props?

Reply:
LinkContainer only uses onClickOutside, I wanted to make sure it doesn't break so I mirrored what popover does: 
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/popover/index.js#L224

I suppose we technically don't need the default for onClickOutside, as popover will handle that for us.